### ISSUE TITLE
CFBundleIdentifier error

### ISSUE THREAD

**Issue by koff75** [infinitered/ignite#1329]
Hello !

---
I would like to start the simulator with the command react-native run-ios : 

```
** BUILD FAILED **
The following build commands failed
Installing build/Build/Products/Debug-iphonesimulator/pizza.app
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=22):
Failed to install the requested application
The bundle identifier of the application could not be determined.
Ensure that the application's Info.plist contains a value for CFBundleIdentifier.
Print: Entry, ":CFBundleIdentifier", Does Not Exist

Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/pizza.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist
```

---
`ignite doctor` results:
```
JavaScript
  node               8.12.0       /usr/local/bin/node  
  npm                6.4.1        /usr/local/bin/npm   
  yarn               1.10.1       /usr/local/bin/yarn  
React Native
  react-native-cli   2.0.1       
  app rn version     0.55.4      
Ignite
  ignite             2.1.2        /usr/local/bin/ignite  
```

Thans 👍 :)

**Comment by AnonymousSB** [comment]
The `CFBundleIdentifier` error is kind of a generic catch all. The real error is most likely listed a few lines above that. I suspect it's the same issue I encountered after upgrading to Mojave and Xcode 10. You'll need to run the following commands, then open Xcode and do a clean

# Commands
```
./node_modules/react-native/scripts/ios-install-third-party.sh
./node_modules/react-native/scripts/ios-configure-glog.sh
```

# Clean build folder
<img width="251" alt="screen shot 2018-10-22 at 9 53 07 am" src="https://user-images.githubusercontent.com/43548615/47315455-5594b600-d5e0-11e8-8461-1d673a5d1797.png">


Then just run `react-native run-ios` again and it should work.

P.S. another issue I've recently encountered with Xcode 10 is that the simulator won't auto launch when running `react-native run-ios`, so I have to use Spotlight to open it manually.

**Comment by koff75** [comment]
Thanks for your answer, but I still have the problem with Xcode 10 and Mojave...

**Comment by AnonymousSB** [comment]
Can you provide more output? I’m interested in the errors right before it says “Build Failed”.

**Comment by gretzky** [comment]
I just dealt with the same issue elsewhere. In order to get any react-native app working on Mojave with Xcode 10, react-native needs to be at v0.57.3. The latest Andross is using 0.55.1. My guess is the issue stems here.

**Comment by kevinvangelder** [comment]
Yeah, unfortunately there are issues with Xcode 10 and React Native. There are a couple of issues here that have some suggested bandaids, but I think a more permanent fix is still coming.

**Comment by jwanga** [comment]
@gretzky this did the trick for me. Had to upgrade to react-native 0.57.4 then clean the build folder from Xcode.

**Comment by gretzky** [comment]
@jwanga, I've already done those steps as well and still experiencing issues, but thanks!

**Comment by kevinvangelder** [comment]
Today's update of 0.57.5 is supposed to resolve the remaining XCode 10 errors. I'd suggest trying to upgrade if you're experiencing issues.

**Comment by esafirm** [comment]
Just try Ignite this morning and have the same issue. 
The one that i created using `react-native init` doesn't have this issue. 

I'm not on Mojave. 

Ignite doctor:

```
JavaScript
  node               10.6.0       /usr/local/bin/node  
  npm                6.4.1        /usr/local/bin/npm   
  yarn               1.10.1       /usr/local/bin/yarn  

React Native
  react-native-cli   2.0.1       
  app rn version     0.57.5      

Ignite
  ignite             2.1.2        /usr/local/bin/ignite  

Android
  java               1.8.0_92     /usr/bin/java               
  android home       -            /usr/local/opt/android-sdk  

iOS
  xcode              10.0      
``` 

Any idea?

**Comment by kevinvangelder** [comment]
@esafirm Did you try the steps posted above by @AnonymousSB? https://github.com/infinitered/ignite/issues/1329#issuecomment-431959264
